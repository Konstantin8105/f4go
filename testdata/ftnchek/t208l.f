      subroutine foo(a,m,n)
      integer m,n
      real a(m,n),b(1,2,3,4,5,6,7)
      end
