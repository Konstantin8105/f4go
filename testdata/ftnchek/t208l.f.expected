SUBROUTINE          	subroutine
IDENT               	foo
(                   	(
IDENT               	a
,                   	,
IDENT               	m
,                   	,
IDENT               	n
)                   	)
NEW_LINE            	

INTEGER             	integer
IDENT               	m
,                   	,
IDENT               	n
NEW_LINE            	

IDENT               	real
IDENT               	a
(                   	(
IDENT               	m
,                   	,
IDENT               	n
)                   	)
,                   	,
IDENT               	b
(                   	(
INT                 	1
,                   	,
INT                 	2
,                   	,
INT                 	3
,                   	,
INT                 	4
,                   	,
INT                 	5
,                   	,
INT                 	6
,                   	,
INT                 	7
)                   	)
NEW_LINE            	

END                 	end
NEW_LINE            	

